README.md

```diff
@@ -1,6 +1,32 @@
 # Product-Portal
 
-## Installation
+## Project Details
+
+### How do we complete this project?
+
+#### Ver 0.1 :
+  - **General concept:** Create a static website using array of objects (30 items - will be made by xx and yy) (each json item: title,price,link,catgeory…)
+    - navbar will include 6~ categories (which will be connected with the json file to right product category)
+    - Search bar input above / beside the navbar (see mockup)
+    - Home/index page will display random 16 items each time with (back - next page at the bottom)
+    - Each li product will display item title > picture > price > button (one below the other in this order)
```

Comment:
ok good question here, click on the product will open a popup baloon with the info a bit bigger more arranged on the screen (again in the popup window) i'll add a mockup to represent it.
about the ver 1.0 i don't want to get into too many details but it will be diffrent ofcourse than from ver 0.1